test/views/authenticated/TaxYearSelectionViewSpec.scala

```diff
@@ -50,7 +50,9 @@ class TaxYearSelectionViewSpec extends CheckboxViewBehaviours[TaxYearSelection]
 
     behave like checkboxPage(form, applyView, messageKeyPrefix, options)
 
-    behave like pageWithBodyText(applyView(form), messageKeyPrefix + ".paragraph")
+    behave like pageWithBodyText(applyView(form),
+      messageKeyPrefix + ".bodyText",
```

Comment:
Can we just make this the message key without any string concatenation?

```suggestion
      "taxYearSelection.bodyText",
```